orderer/consensus/etcdraft/eviction.go

```diff
@@ -139,6 +132,12 @@ func (es *evictionSuspector) confirmSuspicion(cumulativeSuspicion time.Duration)
 	es.halted = true
 	es.logger.Infof("Chain has been halted, pulling remaining blocks up to (and including) eviction block.")
```

Comment:
Please move this:

`es.logger.Infof("Chain has been halted, pulling remaining blocks up to (and including) eviction block.")`

To an else case of the if clause you moved, otherwise this is confusing - it says it is pulling the remaining blocks and then changes its mind about it later on.